ee/ucp/admin/install/install-on-azure.md

```diff
@@ -211,8 +209,7 @@ The `--pod-cidr` option maps to the IP address range that you configured for
 the subnets in the previous sections, and the `--host-address` maps to the
 IP address of the master node.
 
-> Note: The `pod-cidr` range must be within an Azure subnet attached to the
-> host.
+> Note: The `pod-cidr` range must match the Azure Vnet Subnet attached the hosts
```

Comment:
Pedantic but if we're adjust should it read `Azure Virtual Network Subnet`?

Reply:
would apply to more than just subnets actually, as `vnets` is in use throughout doc

Reply:
I've gone through an expanded `vnets` to `Virtual Networks` everywhere I can :)